conda-recipe/meta.yaml

```diff
@@ -14,96 +14,105 @@
 # limitations under the License.
 #===============================================================================
 
-{% set version = '2024.1' %}
+# NB: this recipe should be synced between sklearnex repo and feedstocks
+
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2025.1" %}
 {% set buildnumber = 0 %}
 
 package:
-    name: daal4py
-    version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
+# Reminder: source should point to path for repo or github archive for feedstock
+# source:
+#   url: https://github.com/intel/scikit-learn-intelex/archive/{{ version }}.tar.gz
+#   sha256: ...
 source:
-    path: ..
+  path: ..
 
 build:
-    number: {{ buildnumber }}
-    include_recipe: False
-    script_env:
-    - DPCPPROOT
-    - DALROOT
-    ignore_run_exports:
-       - python
-       - mpich
+  skip: true  # [not (linux64 or win)]
+  number: {{ buildnumber }}
+  include_recipe: False
+  script_env:
+  - DPCPPROOT
+  - DALROOT
+  - NO_DIST=1  # [win]
 
 requirements:
-    build:
-        - {{ compiler('cxx') }}  # [not osx]
-        - {{ compiler('c') }}  # [not osx]
-    host:
-        - python
-        - setuptools
-        - numpy {{ numpy }}
-        - dal-devel ==2024.1.0
-        - cython
-        - jinja2
-        - mpich  # [osx]
-        - impi-devel  # [not osx]
-        - clang-tools
-        - pybind11
-        - make
-    run:
-        - python
-        - dpcpp_cpp_rt ==2024.1.0  # [linux]
-        - dpcpp_cpp_rt ==2024.1.0  # [win or osx]
-        - dal ==2024.1.0
-    ignore_run_exports:
-        - numpy
+  build:
+    - make  # [linux]
+    - dpcpp_linux-64  # [linux64]
+    # - dpcpp_win-64  # [win]
+    - {{ compiler('cxx') }}  # [linux64 or win]
+    - {{ compiler('c') }}  # [linux64 or win]
+    # conda-forge feedstock specific
+    # - {{ stdlib("c") }}  # [linux64 or win]
+  host:
+    - python
+    - setuptools
+    - cmake
+    - clang-format
+    - cython
+    - jinja2
+    - pybind11
+    - numpy {{ numpy }}
+    - mpich  # [not win]
+    # - impi-devel  # [win]
```

Comment:
I don't know much about the mpi-side of things. You have proposed switching the linux support from ```impi``` to ```mpich```.  Could you check with @ethanglaser about these things (out of my knowledge).  Also mpi support for windows is dropped (and is in a comment that may need to be deleted). Is this something to be readded in the future? Also, not to be annoying: Given that dpcpp_cpp_rt was on the ```run:``` side of things, did we have a regression in windows support via this at some point (i.e. did we support dpc building via conda for windows before, or was it just not properly tested, and was erroneously added to ```run:```)?

Reply:
there should be split between GPU mpi support and CPU mpi support.  As well as Intel build of scikit vs public one. 

Overall mpich and mpi are compatibe

Reply:
It's a good point - is there a reason we have mpich here instead of impi_rt/impi-devel?

Reply:
I reverted building and runtime deps to use impi.
Feedstock builds of sklearnex never included dpc part, so there is no regression in support on Windows.